app/views/second_level_browse_page/_new_second_level_browse_pages.html.erb

```diff
@@ -0,0 +1,95 @@
+<%
+  contents = []
+  accordion_contents = []
+  more_on_this_topic_contents = []
+%>
+
+<% page.lists.each_with_index do |list, list_index| %>
+
+  <% contents = capture do %>
+    <%# Add a heading above the A-Z the list. Add it here so that we can extract the heading text %>
+    <% if !curated_order %>
+      <%= render "govuk_publishing_components/components/heading", {
+       text: list["title"],
+       font_size: "m",
+       margin_bottom: 7,
+      }
+     %>
+    <% end %>
+    <%# Render the contents as a list, regardless of whether they'll be in an accordion or as an A-Z list %>
+    <%= render partial: 'new_links', locals: { list: list.contents, list_index: list_index } %>
+  <% end %>
+
+  <%# Generate accordion shaped markup %>
+  <% if curated_order %>
+    <% accordion_items(section_title: list["title"], section_contents: contents, accordion_contents: accordion_contents) %>
```

Comment:
The `accordion_items` helper makes me uneasy as it's not clear what it does and is a bit of a code smell. Whilst it works, is technically okay, and is understandable after a bit of time and effort, I think it signifies that this template is too complicated.

Ideally, this changing of the shape of the data should be done in the controller - then passed into the template. This would also mean that two templates - one for curated order, one for A to Z - could be used to avoid the use of numerous `if` statements inside the template. Whilst this may mean some repetition, it would mean clearer, simpler and more understandable templates.

Reply:
So @hannalaakso and I paired on splitting the second level templates - https://github.com/alphagov/collections/pull/2773 - which should go some way to making this easier to understand and iterate on.

Reply:
This has been resolved with the merging in of #2773 👍